cryptocurrencies-service/src/services/cryptocurrency-service.ts

```diff
@@ -137,6 +139,36 @@ export const getCryptocurrencies = async (limit = 10, offset = 0): Promise<Crypt
     return convertedBindings as Cryptocurrency[];
 };
 
+export const createCryptocurrency = async (cryptocurrency: CreateCryptocurrencyInput): Promise<Cryptocurrency> => {
+    const id = crypto.randomUUID();
```

Comment:
Built-in support for uuid generation
For reference: https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID